### DISCUSSION TITLE
What kind of backend will you use with Tailwind Elements?

### DISCUSSION THREAD

**Discussion by filipkappa** [mdbootstrap/TW-Elements#1445] [Vote in polls]
We'll be creating integration guides soon & possibly separate fully integrated packages in the future.
Let us know what should have the highest priority.

**Update**
Some official integration guides are already published!

- [Express](https://tailwind-elements.com/docs/standard/integrations/express-integration/)
- [Laravel](https://tailwind-elements.com/docs/standard/integrations/laravel-integration/)
- [Django](https://tailwind-elements.com/docs/standard/integrations/django-integration/)

But we're keeping the vote up in order to see what kind of starter packages should we prepare first.

**Comment by killyvera** [comment]
.Next

**Comment by rnandam** [comment]
Your tool i am using in Angular , Thousand thanks to your team I love this product i got what i need

**Reply by tdekoekkoek** [reply]
I'm also using Angular.  I am integrating TE alongside some other Tailwind code

**Comment by BillyAlbornoz** [comment]
Sveltekit, using backend of it with SSR

**Comment by ghost** [comment]
Dioxus or Yew (Rust)

**Comment by gorann1** [comment]
Symfony

**Comment by sunderlands** [comment]
Solidjs, or a Rust web framework maybe

**Comment by fsmutimeer** [comment]
Quarkus

**Comment by sahil-sahu** [comment]
Next.JS

**Comment by deep0133** [comment]
Express

**Comment by LucasDiias** [comment]
NextJS / Fastify

**Comment by abteilung** [comment]
Sanity.io

**Comment by ajayvns97** [comment]
hm

**Comment by Yakumo-Yukari** [comment]
axum (rust)

**Reply by AbhijeetRDarasa** [reply]
Nodejs

**Comment by Rish1327** [comment]
gngng

**Comment by Sansan022** [comment]
Laravel

**Comment by parvezislam45** [comment]
nice

**Comment by SamiAwada** [comment]
Nextjs

**Comment by kevineleven** [comment]
NextJS

**Comment by artonio** [comment]
Solid.js

**Comment by kosirm** [comment]
Just Vite vanilla + capacitor for mobile.

**Comment by amjkwd** [comment]
Gin (golang)

**Comment by DhanBirSunwar** [comment]
ruby on rails (hotwire)
